generators/typescript/utils/core-utilities/reflect/setObjectProperty.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Sets the value at path of object. If a portion of path doesn’t exist it’s created.
+ *
+ * @param object The object to modify.
+ * @param path The path of the property to set.
+ * @param value The value to set.
+ * @return Returns object.
+ */
+export function setObjectProperty<T extends object>(object: T, path: string, value: any): T {
```

Comment:
lets credit lodash if we took it from there ?

Reply:
Oops, totally meant to do that - will add that in.